pyvista/plotting/plotting.py

```diff
@@ -3101,19 +3127,33 @@ def add_mesh(
             specular_power=specular_power,
             show_edges=show_edges,
             color=color,
-            style=style,
+            style=style if style != 'points_gaussian' else 'points',
             edge_color=edge_color,
-            render_points_as_spheres=render_points_as_spheres,
             render_lines_as_tubes=render_lines_as_tubes,
             lighting=lighting,
             line_width=line_width,
             culling=culling,
         )
+
+        if style == 'points_gaussian':
+            self.mapper.emissive = emissive
+            self.mapper.scale_factor = point_size * self.mapper.dataset.length / 1300
+            if not render_points_as_spheres and not emissive:
+                if opacity >= 1.0:
+                    opacity = 0.9999  # otherwise, weird triangles
```

Comment:
Should we report this upstream since it's happening in ParaView as well?

Reply:
Yep, but I'm scared of gitlab :)

Reply:
🤣

Reply:
Not worth raising an issue at this time after a bit of reflection